module/min/mingl/src/camera_orbit_controls.rs

```diff
@@ -195,6 +347,8 @@ crate::mod_interface!
   /// Exposes the `CameraOrbitControls` struct for public use.
   exposed use
   {
-    CameraOrbitControls
+    CameraOrbitControls,
+    CameraState,
+    bind_controls_to_input
   };
```

Comment:
Move bind_controls_to_input under Own
And cameraState is not used outside this file

Add the function bind_controls on Camera from Renderer. So you don't have to call this seperately